content/articles/creation-of-calendar-heatmaps-to-display-time-series-quantitative-data-with-python/index.md

```diff
@@ -0,0 +1,115 @@
+﻿### Creation of Calendar Heatmaps to Display Time-Series Quantitative Data with Python
+In this  simple well-elaborated tutorial we have a lot of alternatives when it comes to choosing a visualization for our data.  
+For a visualization to be said to be  good it's not only easy to do but also has a cognitive and neural basis. A basic heatmap is a graphical representation of data in which individual values are represented by a color gradient.  
+Calendar heatmaps combine heatmaps and calendars into a single visualization. In a calendar view, a calendar heatmap employs colored cells to represent the relative quantity of events on each day.  
+Weeks divide the days into columns, which are then classified by months and years. This allows us to immediately spot daily and weekly patterns as well as to detect anomalies in the data.
+Time series is a design that collects data on the same variable at regular intervals(Days, Weeks, Months, Years).
+It allows the researcher to access the impact of treatment over time, making it easier to predict  future values based on previously observed values. Key areas that implement it are :
+- Business forecasting.
+- Understanding  previous behavior.
+- Planning the future.
+- Evaluate current accomplishments.
```

Comment:
I think this sentence is unnecessary - "In this  simple well-elaborated tutorial we have a lot of alternatives when it comes to choosing a visualization for our data."
Also, use spaces to separate your sentences as they are crowded.